common/namespace/replication_task_executor.go

```diff
@@ -130,6 +131,7 @@ func (h *namespaceReplicationTaskExecutorImpl) shouldProcessTask(ctx context.Con
 	switch err.(type) {
 	case nil:
 		if resp.Namespace.Info.Id != task.GetId() {
+			h.logger.Error("namespace replication encountered name / UUID collision during determination on whether to process task", tag.NewStringTag("namespaceID", task.Info.GetName()), tag.NewErrorTag(err))
```

Comment:
nit: tag.tag.WorkflowNamespaceID or tag.WorkflowNamespace

Reply:
nit: the error is nil at this line, no need to log error

Reply:
thanks! fixed both